lib/tom_queue/deferred_work_manager.rb

```diff
@@ -80,13 +80,19 @@ def schedule(response, headers, payload)
     def start
       debug "[DeferredWorkManager] Deferred process starting up"
 
+      register_signal_handlers
```

Comment:
This doesn't belong in here - it's specific to the daemon, rather than a deferred work manager class.